### DISCUSSION TITLE
Can't install Fortawesome/react-fontawesome

### DISCUSSION THREAD

**Discussion by rodneybejabeng** [FortAwesome/Font-Awesome#19225] [Q&A]
when I run npm install --save @fortawesome/react-fontawesome to my project directory I get this error message:
npm ERR! code E401
npm ERR! Incorrect or missing password.
npm ERR! If you were trying to login, change your password, create an
npm ERR! authentication token or enable two-factor authentication then
npm ERR! that means you likely typed your password in incorrectly.
npm ERR! Please try again, or recover your password at:
npm ERR!     https://www.npmjs.com/forgot
npm ERR! 
npm ERR! If you were doing some other operation then your saved credentials are
npm ERR! probably out of date. To correct this please try logging in again with:
npm ERR!     npm login

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bigrod/.npm/_logs/2022-08-01T17_55_06_397Z-debug-0.log

my terminal recognizes that I'm already logged in with the correct credentials so I'm not sure what the problem is. I'm unable to install over font awesome library also. 

<img width="613" alt="Screen Shot 2022-08-01 at 1 09 12 PM" src="https://user-images.githubusercontent.com/107449260/182218155-caa4f394-0f2f-4a8e-a713-9335dabcb043.png">

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Sorry but this is hard to debug

If you are a Font Awesome Pro customer, please send an email to hello@fontawesome.com
